Cargo.toml

```diff
@@ -42,4 +43,6 @@ anyhow = "1.0.71"
 ethers = { version = "2.0.7", features = ["abigen"] }
 
 [dev-dependencies]
-mockito = "1.1.0"
+diesel = { version = "2.1.0", features = ["postgres", "r2d2", "serde_json"] }
+diesel_migrations = { version = "2.1.0", features = ["postgres"] }
```

Comment:
Is it desired to have `diesel` and `diesel_migrations` both in `[dependencies]` and `[dev-dependencies]`?

Reply:
Yes as we use it in both the app and in the integration tests that interact with the database.